topics/genome-annotation/tutorials/hpc-for-lsgc/tutorial.md

```diff
@@ -0,0 +1,400 @@
+- Learn how to run different tools in Galaxy to perform sequence comparison at fine and coarse-grained levels
+- Learn how to post-process your sequence comparisons
+time_estimation: 2H
+key_points:
+- We learnt that sequence comparison is a demanding problem and that there are several ways to approach it
+- We learnt how to run sequence comparisons in Galaxy with different levels of precision, particularly fine-grained and coarse-grained sequence comparison using GECKO and CHROMEISTER for smaller and larger sequences, respectively
+- We learnt how to post-process our comparison by extracting alignments, fine-tuning with multiple sequence alignment, detecting rearrangements, etc.
+requirements:
+  -
+    type: "internal"
+    topic_name: introduction
+    tutorials:
+      - galaxy-intro-101
+
+contributors:
+- estebanpw
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+Sequence comparison is a core problem in bioinformatics. It is used widely in evolutionary studies, structural and functional analyses, assembly, metagenomics, etc. Despite its regular presence in everyday Life-sciences pipelines, it is still not a trivial step that can be overlooked. Therefore, understanding how sequence comparison works is key to developing efficient workflows that are central to so many other disciplines.
+
+In the following tutorial, we will learn how to compare both small and large sequences using both seed-based alignment methods and alignment-free methods, and how to post-process our comparisons to refine our results. Besides, we will also learn about the theoretical background of sequence comparison, including why some tools are suitable for some jobs and others are not.
+
+This tutorial is divided into two large sections:
+
+ - Fine-grained sequence comparison: In this part of the tutorial we will use `GECKO` ({% cite GECKO %}) to perform sequence alignment between small sequences. We will also identify, extract and re-align regions of interest.
+ - Coarse-grained sequence comparison: In this part of the tutorial, we will tackle on how to compare massive sequences using `CHROMEISTER` ({% cite perez2019ultra %}), an alignment-free sequence comparison tool. We will generate visualization plots for the comparison of large plant genomes and automatically detect large-scale rearrangements.
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Fine-grained sequence comparison
+
+Imagine you are working on an evolutionary study regarding the species `mycoplasma hyopneumoniae`. In particular, you are interested in the strains `232` and `7422` and wish to compare their DNA sequence to know more about the evolutionary changes that took place between both. The workflow you will follow starts with (1) acquiring the data, (2)  getting it ready for Galaxy, (3) running the comparison and (4) inspecting and working with the resulting alignments. Let's go! 
```

Comment:
_Mycoplasma hyopneumoniae_ (cursive and uppercase). It is necessary to Italicize genus and species names.

Reply:
I am checking all instances of this, but I found that the italics dont work in the legend of a figure, do they?
In particular: 
`![Mycoplasma hyopneumoniae comparison example](../../images/hpc-for-lsgc/GeckoMGV01.PNG "(Left) Sequence comparison between Mycoplasma hyopneumoniae 232 and 7422. (Right) Zoomed-in region where the repeats of interest are located.")`

Any idea?

Reply:
This is a known bug I think. you can do <em>Mycoplasma hyopneumoniae</em> in the figure caption.

Reply:
I tried this:
`![Mycoplasma hyopneumoniae comparison example](../../images/hpc-for-lsgc/GeckoMGV01.PNG "(Left) Sequence comparison between *Mycoplasma hyopneumoniae* 232 and 7422. (Right) Zoomed-in region where the repeats of interest are located.")` but it renders like with the asterisks included:

![imagen](https://user-images.githubusercontent.com/4108045/106867113-81fb7000-66cd-11eb-8f53-6aa1a2a88a27.png)

Reply:
Yes, please do `<em>Mycoplasma hyopneumoniae</em>`

Reply:
Thanks! That worked perfectly